modules/core/test/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+include_directories(${Core_INCLUDE_DIR})
```

Comment:
Core_INCLUDE_DIR isn't set here, but it looks like the reason this still works is that CMake actually inherits include directories from its parent CMakeLists as long as you call include_directories before adding the subdirectory.